ha/setup/v2/ha_setup.py

```diff
@@ -0,0 +1,233 @@
+#!/bin/env python3
+
```

Comment:
Add entry in pyinstaller

check
jenkins/pyinstaller/v2/pyinstaller-cortx-ha.spec
jenkins/rpm/v2/cortx-ha.spec
from https://github.com/Seagate/cortx-ha/pull/279/files

Reply:
done